wallets/rn_cli_wallet/src/components/CircleActionButton.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {TouchableOpacity, StyleSheet, Image} from 'react-native';
+import LinearGradient from 'react-native-linear-gradient';
+
+interface CircleActionButtonProps {
+  copyImage: boolean;
+  handlePress: () => void;
+}
+
+const clipboardImage = require('../assets/Copy.png');
+const QRCodeImage = require('../assets/QRCode.png');
```

Comment:
i prefer to use `import`. Is it showing an error with png files? If so, adding [this](https://github.com/WalletConnect/react-native-examples/blob/main/dapps/v2Explorer/declarations.d.ts) should work